applicationdevelopment/assets/css/style.css

```diff
@@ -0,0 +1,120 @@
+body {
+	font-family: Myriad Pro;
+	color: #666666;
+	padding: 0;
+	font-size: 14px;
+}
+* {
+	box-sizing: border-box;
+}
+.clearfix:after {
+	visibility: hidden;
+	display: block;
+	font-size: 0;
+	content: " ";
+	clear: both;
+	height: 0;
+}
+* html .clearfix             { zoom: 1; } /* IE6 */
```

Comment:
I would like to see this on your PC.
And if you want to target IE specifically use like this
<!--[if lt IE 7]>  <html class="ie ie6 lte9 lte8 lte7"> <![endif]-->

and in css
.ie.ie6 .clearfix{

}